suites/squid/cephfs/tier-0_cephfs_mirrror_upgrade_ibm_7x_to_8x.yaml

```diff
@@ -174,6 +164,17 @@ tests:
       module: cephfs_mirroring.snapdiff_perf_improvements_across_releases.py
       name: Validate snapshot synchronisation perf improvements
       polarion-id: "CEPH-83595260"
+  - test:
```

Comment:
Can you please check if other upgrade suites that runs this test also needs re-organising of tests. 
tier-0_cephfs_mirrror_upgrade_6x_to_8x.yaml
tier-0_cephfs_mirrror_upgrade_7x_to_8x.yaml
tier-0_cephfs_mirrror_upgrade_ibm_6x_to_8x.yaml
tier-0_cephfs_mirrror_upgrade_seq_6x_to_8x.yaml